systemtest/src/test/java/io/strimzi/systemtest/report/DefaultClusterReportST.java

```diff
@@ -0,0 +1,448 @@
+        assertValidYamls(outPath + "/reports/kafkarebalances",
+            KafkaRebalance.class, clusterName + ".yaml", 1);
+    }
+
+    private void assertValidKafkas(String outPath, String clusterName) throws IOException {
+        for (String s : Arrays.asList(
+            clusterName + ".yaml",
+            clusterName + "-tgt.yaml"
+        )) {
+            assertValidYamls(outPath + "/reports/kafkas", Kafka.class, s, 1);
+        }
+    }
+
+    private void assertValidKafkaTopics(String outPath) throws IOException {
+        // skipping internal topics as they will not be visible with the UTO by default
+        assertValidYamls(outPath + "/reports/kafkatopics", KafkaTopic.class, "my-topic.yaml", 1);
+    }
+
+    private void assertValidKafkaUsers(String outPath) throws IOException {
+        // skipping internal topics as they will not be visible with the UTO by default
+        assertValidYamls(outPath + "/reports/kafkausers", KafkaUser.class, "my-user.yaml", 1);
+    }
+
+    private void assertValidStrimziPodSets(String outPath, String clusterName) throws IOException {
+        for (String s : Arrays.asList(
+            clusterName + "-kafka.yaml",
+            clusterName + "-tgt-kafka.yaml",
+            clusterName + "-tgt-zookeeper.yaml",
+            clusterName + "-zookeeper.yaml"
+        )) {
+            assertValidYamls(outPath + "/reports/strimzipodsets", StrimziPodSet.class, s, 1);
+        }
+    }
+
+    private void assertValidConfigs(String outPath, String clusterName) {
+        for (String s : Arrays.asList(
+            clusterName + "-kafka-0.cfg",
+            clusterName + "-kafka-1.cfg",
+            clusterName + "-kafka-2.cfg",
+            clusterName + "-zookeeper-0.cfg",
+            clusterName + "-zookeeper-1.cfg",
+            clusterName + "-zookeeper-2.cfg")
+        ) {
+            assertValidFiles(outPath + "/reports/configs", s, 1);
+        }
+    }
+
+    private void assertValidLogs(String outPath, String clusterName) {
+        assertValidFiles(outPath + "/reports/logs", "events.txt", 1);
+        for (String s : Arrays.asList(
+            clusterName + "-kafka-0.log",
+            clusterName + "-kafka-1.log",
+            clusterName + "-kafka-2.log",
+            clusterName + "-zookeeper-0.log",
+            clusterName + "-zookeeper-1.log",
+            clusterName + "-zookeeper-2.log"
+        )) {
+            assertValidFiles(outPath + "/reports/logs", s, 1);
+        }
+        assertValidFiles(outPath + "/reports/logs", clusterName + "-cruise-control", 1);
+        assertValidFiles(outPath + "/reports/logs", "my-bridge-bridge", 1);
+        assertValidFiles(outPath + "/reports/logs", clusterName + "-entity-operator", 3);
+        assertValidFiles(outPath + "/reports/logs", "my-connect-connect", 1);
+        assertValidFiles(outPath + "/reports/logs", "my-mm2-mirrormaker2", 1);
+        assertValidFiles(outPath + "/reports/logs", "strimzi-cluster-operator", 2);
```

Comment:
Maybe you could group the one with num=1 and put them in the for loop as other tests?

Reply:
Nope, as they leverage the prefix logic here (no stable identities in standard deploy).

Reply:
That is due to my previous implementation, I'll bring them inside the loop.